impls/monero.rs/example/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "monero_example"
+version = "0.0.1"
+edition = "2021"
+
+[dependencies]
+monero_rust = { path = ".." }
```

Comment:
```suggestion
monero_rust = { path = ".." }
# monero_rust = { git = "https://github.com/MrCyjaneK/monero_c" }
```

According to rust docs it should be able to find the dependency on it's own in the monero_c monorepo.